webpage/github-pandoc.css

```diff
@@ -262,7 +275,7 @@ sup {
 }
 
 sup {
-    top: -0.5em;
+    top: -0.1em;
```

Comment:
Will this fix the superscript appearance issues described at https://github.com/greenelab/meta-review/pull/88#issuecomment-408546057?

Reply:
No, that's just to make it look a little better (in my judgement). The real culprit of the problem was this:

https://github.com/greenelab/manubot-rootstock/pull/159/commits/ba17b84e173e2d3406688791fece39db2efd6ff0#diff-f52f320f93db27d62530fdd59fbcfb06L788